e2e/tests/datasets/dataset-export-r-notebook.spec.ts

```diff
@@ -12,7 +13,7 @@ describe('Export Dataset to Notebook Test', () => {
     await signInWithAccessToken(page);
   });
 
-  const workspaceName = 'e2eDatasetExportToRNotebookTest';
+  const workspaceName = makeWorkspaceName();
```

Comment:
To be more explicit, we can remove this line and change L23 to `await createWorkspace(page);`

Reply:
Unfortunately I have to revert this change because this test assert workspaceName in 2 places below